pkg/objstore/swift/swift.go

```diff
@@ -0,0 +1,314 @@
+/*
+Copyright 2018 eBay Inc.
```

Comment:
Can we remove the licence comment to keep in line with rest of the repo

Reply:
Removed.